frontend/packages/kubevirt-plugin/src/selectors/vm-template/basic.ts

```diff
@@ -65,8 +65,9 @@ export const getTemplateProvider = (
   withProviderPrefix = false,
 ): string => {
   let provider = getAnnotation(template, TEMPLATE_PROVIDER_ANNOTATION);
-  if (!provider && isCommonTemplate(template)) {
-    const isUpstream = window.SERVER_FLAGS.branding === 'okd';
+  const isUpstream = window.SERVER_FLAGS.branding === 'okd';
```

Comment:
We have this exact same test in multiple places, including two different locations in this PR. I think we should extract it out to a function somewhere like frontend/packages/kubevirt-plugin/src/utils/common.ts so we can just call it wherever we need it.

Reply:
https://github.com/openshift/console/pull/8918